tests/dummy/app/helpers/i.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+const matcher = /\$\{\s*(.*?)\s*\}/g;
+
+export function interpolate([ str ], hash) {
```

Comment:
I didn't want to add a package dependency w/o asking first. If you want to move the demo to sparkles I can just remove that helper.